### PR TITLE
[Feature: #163863687] Add Cover Image Url to Article Model

### PR DESCRIPTION
**Pull request by ebenezerdon**
#### What does this PR do?
This PR adds a cover image `URL` column to the `Article Model` and `controller`

#### Description of Task to be completed?
- Users should be able to add a cover image when creating an article

#### How should this be manually tested?
- run `db:migrate`
- the tab for `coverImageUrl` should be on your `Article` table
- you can now add a `coverImageUrl` param to `POST` request for articles

#### What are the relevant pivotal tracker stories?
[ #163863687](https://www.pivotaltracker.com/story/show/163863687)

### CODE REVIEW

File: controllers/article.js [hunk 1]
```diff
@@ -24,9 +24,16 @@ const {
 export default {
   async createArticle(req, res) {
     const {
-      title, body, description, category,
+      title, body, description, category, coverImageUrl
     } = req.body;
 
+    let imgUrl;
```
Comment (oreoluwade) [inline controllers/article.js]: What are you trying to do here? Why the change of variables?

Comment (ebenezerdon) [reply controllers/article.js]: I had to give `coverImageUrl` a default value because of the `trim()` method when sending a post request. It fails because `Null` value cannot be trimmed.

Comment (oreoluwade) [reply controllers/article.js]: In that case, instead of this, on line 51, you could do:

`coverImageUrl: '' || coverImageUrl.trim()`

Comment (ebenezerdon) [reply controllers/article.js]: Thanks, I've tried this but there's an error when I try to create an article without the `coverImageUrl`:
`Cannot read property 'trim' of undefined`

Comment (oreoluwade) [reply controllers/article.js]: `coverImageUrl: ''coverImageUrl && coverImageUrl.trim() || ''`

Comment (ebenezerdon) [reply controllers/article.js]: Thanks, I've made the change.

**Review by oreoluwade — APPROVED** [review]
